### ISSUE TITLE
Add possibility to assign "cloudsigma_ssh_key" to "cloudsigma_server"

### ISSUE THREAD

**Issue by wojcik92michal** [cloudsigma/terraform-provider-cloudsigma#29]
# Feature request

### Description
According to the docs (version `1.6.0`, https://registry.terraform.io/providers/cloudsigma/cloudsigma/latest/docs/resources/server), seems to me that it is not possible to attach the SSH keys to the server. It is it a little inconvenient to assign manually SSH keys which are created by the Terraform Provider.

### New or Affected Resource(s)

* cloudsigma_server

### Potential Terraform Configuration

```hcl
resource "cloudsigma_server" "web" {
  cpu          = 2000              # 2GHz CPU
  memory       = 512 * 1024 * 1024 # 512MB RAM
  name         = "web"
  vnc_password = "cloudsigma"

  ssh_keys = [
    "first-ssh-key-uuid",
    "second-ssh-key-uuid",
  ]
}
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
  to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by pavel-github** [comment]
👋 @wojcik92michal, it looks like our documentation didn't contain this info. Assigning SSH keys are possible: https://github.com/cloudsigma/terraform-provider-cloudsigma/blob/master/cloudsigma/resource_cloudsigma_server.go#L100-L107.
Here is an [example](https://github.com/pavel-github/k3s-cluster-demo/blob/master/k3s-server/main.tf#L42) of working configuration.

Thanks for spotting it 👍. Do you mind to create a PR for it?

**Comment by pavel-github** [comment]
The fix commit: 74c3adfed776422e8e98963a54555a60cc514f63.
The documentation will be published to the Terraform Registry with the next release (v1.6.1).

**Comment by pavel-github** [comment]
The release is available in Terraform Registry: https://registry.terraform.io/providers/cloudsigma/cloudsigma/1.7.2.
